content/projects/tdd/string-calculator/_index.md

```diff
@@ -117,6 +117,15 @@ In the first case, the default delimiter is: ";".
 
 In the second case, the default delimiter is: "4".
 
+**Note:** you are not expected to handle integer delimiters where the delimiter and the digit your adding are the same. For example
+
+```
+add("//4\n14244")
+add("//88\n18882")
+```
```

Comment:
can you also add some more examples on the invalid section

Reply:
Maybe useful to show examples of what we do expect them to be able handle?